### PR TITLE
Highlighted the batch who is working from office today

### PR DESCRIPTION
**Pull request by tejaszambre**

### CODE REVIEW

**Comment by tejaszambre** [comment]
[Highlight WFO batch name](https://trello.com/c/mOyNiAtz/52-highlight-wfo-batch-name)

**Comment by tejaszambre** [comment]
[Accessibility Issues](https://trello.com/c/OBB1bAV9/40-accessibility-issues)

File: app/helpers/projects_helper.rb [hunk 1]
```diff
@@ -11,4 +11,8 @@ def active_project_ids(user)
   def get_users_projects(user)
     user.user_projects.where(active: true).nin(project: @project.id)
   end
+
+  def alpha_week?
```
Comment (Ashish944) [inline app/helpers/projects_helper.rb:15]: Fix the method name

File: app/views/projects/_list.html.haml [hunk 2]
```diff
@@ -21,7 +21,11 @@
           %td= project.managers.approved.collect(&:name).join(', ')
         - else
           %td= 'NA'
-        %td= project.try(:batch_name) || 'NA'
+        - wfo_batch_name = alpha_week? ? 'Alpha' : 'Bravo'
+        - if project.try(:batch_name) == wfo_batch_name
+          %td{style: 'color: red;'}= project.try(:batch_name) || 'NA'
```
Comment (Ashish944) [inline app/views/projects/_list.html.haml:26]: No need for or condition || 'NA'. Also add external css

File: app/views/projects/show.html.haml [hunk 3]
```diff
@@ -32,7 +32,11 @@
           %td= link_to @project.company.try(:name), @project.company
         %tr
           %th Batch Name
-          %td= @project.batch_name
+          - wfo_batch_name = alpha_week? ? 'Alpha' : 'Bravo'
+          - if @project.try(:batch_name) == wfo_batch_name
+            %td{style: 'color: red;'}= @project.try(:batch_name) || 'NA'
```
Comment (Ashish944) [inline app/views/projects/show.html.haml:37]: No need for or condition || 'NA'. Also add external css

File: app/views/projects/show.html.haml [hunk 4]
```diff
@@ -32,7 +32,11 @@
           %td= link_to @project.company.try(:name), @project.company
         %tr
           %th Batch Name
-          %td= @project.batch_name
+          - wfo_batch_name = alpha_week? ? 'Alpha' : 'Bravo'
```
Comment (Ashish944) [inline app/views/projects/show.html.haml:35]: get the wfh batch name before this each loop or in action method

File: app/helpers/projects_helper.rb (hunk 1 above)
Comment (tejaszambre) [reply app/helpers/projects_helper.rb:15]: yes okay

File: app/views/projects/_list.html.haml (hunk 2 above)
Comment (tejaszambre) [reply app/views/projects/_list.html.haml:26]: Okay

File: app/views/projects/show.html.haml (hunk 3 above)
Comment (tejaszambre) [reply app/views/projects/show.html.haml:37]: Okay

File: app/views/projects/show.html.haml (hunk 4 above)
Comment (tejaszambre) [reply app/views/projects/show.html.haml:35]: Okay

**Review by Ashish944 — APPROVED** [review]

**Comment by Sameer-Tilak** [comment]
LGTM

**Pull request merged**